dataland-frontend/tests/e2e/specs/low/pages/taxonomy/Paginator.ts

```diff
@@ -0,0 +1,25 @@
+import Chainable = Cypress.Chainable;
+
+describe('Paginator test suite', () => {
+    beforeEach(() => {
+        cy.restoreLoginSession()
+    })
+    it('Assure existence of paginator for Dax search', () => {
+        cy.visit('/searchtaxonomy')
+        cy.get('#app').should("exist")
```

Comment:
Gleiche Frage wie oben, braucht man das immer?

Reply:
Müssten wir Stephan Sinn theoretisch fragen, was sein Gedanke dabei war.
Im Grunde checkst du ja damit, ob die App gemountet ist.

Andererseits checkt man ja in der nächsten Zeile:

`cy.get('h1').should("contain", "Search EU Taxonomy data")`

Und wenn dieser Check erfolgreich ist, heißt das ja gleichzeitig auch, dass die App gemountet sein muss.


Ich denke ich würde es einfach drin lassen.
Dann checkt man quasi mit jeder Zeile nur eine Tatsache.

Reply:
siehe oben

Reply:
siehe ebenfalls oben.   leider jedoch in den Overlay Branch gepusht

Reply:
cherry-pick durchgeführt

Reply:
ok